src/main/scala/scala/slick/driver/JdbcActionComponent.scala

```diff
@@ -174,20 +174,26 @@ trait JdbcActionComponent extends SqlActionComponent { driver: JdbcDriver =>
   def createSchemaActionExtensionMethods(schema: SchemaDescription): SchemaActionExtensionMethods =
     new SchemaActionExtensionMethodsImpl(schema)
 
-  class SchemaActionExtensionMethodsImpl(schema: SchemaDescription) extends super.SchemaActionExtensionMethodsImpl {
-    def create: DriverAction[Effect.Schema, Unit] = new DriverAction[Effect.Schema, Unit] with JdbcDatabaseAction[Unit] {
-      def statements = schema.createStatements
+  private def mkAction[E <: Effect](dumpInfoKey: String, _statements: Iterator[String])
```

Comment:
should probably be rolled into SimpleJdbcAction (which is not a SqlAction at the moment)